demo/handoff/corkboard.js

```diff
@@ -0,0 +1,33 @@
+// Copyright (C) 2019 Agoric, under Apache License 2.0
+
+import harden from '@agoric/harden';
+
+// Allows multiple parties to store values for retrieval by others.
+function makeCorkboard(name) {
+  const namedEntries = new Map();
+  const orderedEntries = [];
```

Comment:
What is orderedEntries used for? I see there's a todo to retrieve by numbered position, I assume that orderedEntries.length gives that numbered position, is that right? would a count give the same functionality?

Reply:
Dean suggested this functionality. I think part of the idea was to enable displaying a transcript of the stuff exchanged between two parties. Since I'm not sure how this would get used, I only built a tiny bit of it, I'm happy to rip it out, leave it as is, or add the missing functionality.